immudb/dataconverter.py

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
minor: the license should be added here right?

Reply:
Resolved in #50e437a